### PR TITLE
Bump GA Versions for 6.0 release

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/27547

## Description
Bumping to 6.0, dropping 4.1

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -530,7 +530,7 @@ jobs:
                   fi
             - *store_artifacts
       - *persist_to_workspace
-  Server 4_1:
+  Server 4_5:
```
Comment (yaakovi) [inline .circleci/config.yml:533]: I think it will be better to revert the changes in this file and just remove the 4_1 job and create a fresh one.
each job has it's own settings, for example:
`Server 6_0` should have the upload packs to production steps.
`Server Master` should have the option of running build without AMI (special builds).

**Review by yaakovi — CHANGES_REQUESTED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -813,7 +847,7 @@ jobs:
                       python3 ./Tests/scripts/wait_until_server_ready.py "Demisto"
```
Comment (yaakovi) [inline .circleci/config.yml:847]:
```suggestion
                      python3 ./Tests/scripts/wait_until_server_ready.py "master"
```

**Review by yaakovi — APPROVED** [review]

**Review by yuvalbenshalom — APPROVED** [review]
